ietf/templates/doc/document_info.html

```diff
@@ -11,15 +11,15 @@
         <th scope="row">{% if document_html %}Document type{% else %}Type{% endif %}</th>
         <td class="edit"></td>
         <td>
-            {% if doc.get_state_slug == "rfc" and not snapshot %}
+            {% if doc.is_rfc %}
```

Comment:
Ok - I'd started on this road and saw you were going the type route. Decided to stick with this for this PR rather than mixing things up. I'll add switching these to the todo list.